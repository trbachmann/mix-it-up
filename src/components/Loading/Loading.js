import React from 'react';
import mixingImg from '../../images/charisse-kenion-unsplash.jpg';

export const Loading = () => {
  return(
    <div className='Loading'>
      <h1>Mixing up some great recipes...</h1>
      <img
        className='Loading--img' 
        src={mixingImg} 
        alt='chocolate chip cookie dough in a mixing bowl'/>
      <p className='Loading--p--credit'>Photo by Charisse Kenion on Unsplash</p>
    </div>
  );
}