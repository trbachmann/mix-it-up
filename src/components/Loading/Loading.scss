.Loading {
  text-align: center;
}

.Loading--img {
  border-radius: 8px;
  display: block;
  margin: auto;
  width: 50%;
}

.Loading--p--credit {
  font-size: .8em; 
  margin-top: .8em;
}