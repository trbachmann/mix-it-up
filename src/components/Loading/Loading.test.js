import React from 'react';
import { Loading } from './Loading';
import { shallow } from 'enzyme';

describe('Loading', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});