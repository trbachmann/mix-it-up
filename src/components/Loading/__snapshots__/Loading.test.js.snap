// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the snapshot 1`] = `
<div
  className="Loading"
>
  <h1>
    Mixing up some great recipes...
  </h1>
  <img
    alt="chocolate chip cookie dough in a mixing bowl"
    className="Loading--img"
    src="charisse-kenion-unsplash.jpg"
  />
  <p
    className="Loading--p--credit"
  >
    Photo by Charisse Kenion on Unsplash
  </p>
</div>
`;
